docs/sources/docker-hub-enterprise/adminguide.md

```diff
@@ -1,104 +0,0 @@
-no_version_dropdown: true
```

Comment:
Guessing from the removal of DHE here, that they will be moved somewhere else;

How are SEO redirects handled now that `s3_website.json` is no longer there?